src/functions/durableOrchestration.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpRequest, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableOrchestrationClient, IOrchestrationFunctionContext } from 'durable-functions/lib/src/classes';
+
+df.orchestration('DurableFunctionsOrchestratorJS', function* (context: IOrchestrationFunctionContext) {
+    const outputs = [];
+    outputs.push(yield context.df.callActivity('Hello', 'Tokyo'));
+    outputs.push(yield context.df.callActivity('Hello', 'Seattle'));
+    outputs.push(yield context.df.callActivity('Hello', 'Cairo'));
+
+    return outputs;
+});
+
+df.activity('Hello', (context: InvocationContext, name: string) => {
+    return `Hello, ${name}`;
+});
+
+df.httpClient(
+    'DurableFunctionsHttpStart',
+    async (context: InvocationContext, req: HttpRequest, client: DurableOrchestrationClient) => {
```

Comment:
Please use `request` instead of `req` to be consistent with other functions. It's not that much longer and people thought `req` might mean `require` in the user study.

Reply:
Done